src/stylesheets/components/_footer.scss

```diff
@@ -4,13 +4,13 @@
     list-style-type: none;
     margin-bottom: 1em;
 
-    @media (min-width: $breakpoint-small) {
+    @media (min-width: $breakpoint-medium) {
```

Comment:
are these from uikit? I should start using these more...

Reply:
yes! these are supplied by uikit, they are super handy.